docs/source/private/overview.rst

```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
+
+
+.. code-block:: bash
+
+   # If you don’t have cofactors
+   python plan_rbfe_network.py --pdb protein.pdb --ligands ligands.sdf --output network_setup
+
+   # If you have cofactors
+   python plan_rbfe_network.py --pdb protein.pdb --ligands ligands.sdf --cofactors cofactors.sdf --output network_setup
+
+
+You should then execute each transformation using the `quickrun method <https://docs.openfree.energy/en/latest/guide/execution/quickrun_execution.html>`_.
+Below is an example script that will create and submit each job to a SLURM cluster scheduler:
+
+
+.. code-block:: bash
+
+   for file in network_setup/transformations/*.json; do
+     relpath="${file:30}"  # strip off "network_setup/"
+     dirpath=${relpath%.*}  # strip off final ".json"
+     jobpath="network_setup/transformations/${dirpath}.job"
+     if [ -f "${jobpath}" ]; then
+       echo "${jobpath} already exists"
+       exit 1
+     fi
+     for repeat in {0..2}; do
+       cmd="openfe quickrun ${file} -o results_${repeat}/${relpath} -d results_${repeat}/${dirpath}"
+       echo -e "#!/usr/bin/env bash\n${cmd}" > "${jobpath}"
+       sbatch "${jobpath}"
+     done
+   done
+
+
+My Simulations Are Failing, What Do I Do?
+=========================================
+
+
+Unfortunately for these benchmarks, the OpenFE team will only be able to provide help in a
+limited manner (i.e. we will not be able to look at your structures).
+
+To help you, we have created a :ref:`preparing and debugging simulations FAQ <prep_and_debug>` with
+some common issues you may encounter.
+
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   prep_and_debug
+
+
+Cleaning Results
+================
+
+
+.. note::
+   Please keep all post-cleanup data around for analysis until the end of the benchmarks (i.e.
+   after publication).
+
+
+The OpenFE tools are known to generate a lot of data by default (something we are looking to fix!).
+
+We recommend that folks use the :ref:`simulation cleanup <post-simulation cleanup>` script to clean up
+unecessary data.
+
+
+Inspecting Results
+==================
+
+
+.. note::
+   A separate script will be provided for gathering relevant FE output data in Phase 2.
+   This is not how all the data that OpenFE will gather.
+
+
+If you wish to look at your results, you can use the `extract_results.py` script used in the
+public dataset benchmarks:
+
+
+.. code-block:: bash
+
+   wget https://raw.githubusercontent.com/OpenFreeEnergy/IndustryBenchmarks2024/main/industry_benchmarks/utils/extras/extract_results.py
+   python extract_results.py
+
+
+This will provide both dG and ddG outputs for you to further manipulate.
+
+As we cannot tell what format your experimental results are in, we do not provide a plotting script at this time
+and encourage you to use your own internal plotting tools. Note: this may change depending on demand.
```

Comment:
Maybe we shouldn't encourage them to request more scripts?

Reply:
Removed.